ntoskrnl/mm/freelist.c

```diff
@@ -537,20 +537,19 @@ ULONG
 NTAPI
 MmGetReferenceCountPage(PFN_NUMBER Pfn)
```

Comment:
I see a bunch of callers of this function that don't acquire the PFN lock, e.g. `MmpPageOutPhysicalAddress` acquires a fast mutex after the call so is clearly not holding a spin lock.
Might be NewCc only or something, but we probably should update anything that grep shows.

Reply:
Yes this is NewCC. I overlooked this. Maybe it's time to remove it altogether anyway...